src/main/kotlin/org/jitsi/rtp/rtp/RtpPacket.kt

```diff
@@ -208,7 +208,7 @@ open class RtpPacket(
         // The bytes in the header extensions, excluding the (0xBEDE, length)
         // field and any padding.
         var extensionBytes = 0
-        if (hasExtensions) {
+        if (extensionBit) {
```

Comment:
interesting, was this an issue because something had moved at this point, so reading `hasExtensions` directly was reading the wrong thing?

Reply:
This one is actually not needed, but I figured I'd use the value that I already have. The one below (line 234) is the one that caused a problem. When we access it we have already moved the header to offset 0. In the case of Octo, the original offset was 8, which means that we have overriden the original first byte with the first byte of the SSRC (that's why I changed the SSRC value to trigger the bug)

Reply:
👍